addons/sourcemod/scripting/surftimer/sql.sp

```diff
@@ -3458,13 +3441,388 @@ public void db_deleteCheckpoints()
+		{
+			if (IsValidClient(i) && !IsFakeClient(i) && !g_bSettingsLoaded[i] && !g_bLoadingSettings[i])
+			{
+				szSteamID = "";
+				GetClientAuthId(i, AuthId_Steam2, szSteamID, 32, true);
+				g_iSettingToLoad[i] = 0;
+				LoadClientSetting(i, g_iSettingToLoad[i]);
+				g_bLoadingSettings[i] = true;
+				break;
+			}
+		}
+
+		float time = g_fTick[client][1] - g_fClientsLoading[client][0];
+		char szName[32];
+		GetClientName(client, szName, sizeof(szName));
+		LogToFileEx(g_szLogFile, "[SurfTimer] Finished loading %s - %s settings in %fs", g_szSteamID[client], szName, time);
+	}
+
+	return;
+}
+
+public void db_bonusprinforuntimecallback(Handle owner, Handle hndl, const char[] error, DataPack pack)
+{
+
+	ResetPack(pack);
+	int client = ReadPackCell(pack);
+
+	char szSteamID[32];
+	ReadPackString(pack, szSteamID, 32);
+
+	char szMapName[32];
+	ReadPackString(pack, szMapName, 32);
+
+	if (hndl == null)
+	{
+		LogError("[SurfTimer] SQL Error (db_prinforuntimecallback): %s", error);
+		CloseHandle(pack);
+		return;
+	}
+
+	if (!IsValidClient(client)){
+		CloseHandle(pack);
+		return;
+	}
+
+	if(SQL_HasResultSet(hndl) && SQL_FetchRow(hndl)){
+		//PrintToConsole(client,"ALREADY EXISTS");
+
+		float runtime;
+		int zonegroup;
+
+		// Escape name for SQL injection protection
+		char szName[MAX_NAME_LENGTH * 2 + 1], szUName[MAX_NAME_LENGTH];
+		GetClientName(client, szUName, MAX_NAME_LENGTH);
+		SQL_EscapeString(g_hDb, szUName, szName, MAX_NAME_LENGTH);
+		
+		char szQuery[1024];
+		runtime = SQL_FetchFloat(hndl, 0);
+		zonegroup = SQL_FetchInt(hndl, 1);
+
+		Format(szQuery, 1024, sql_insertPR, szSteamID, szName, szMapName, runtime, zonegroup, 0.0, 0.0, 0.0, 0.0);
+		SQL_TQuery(g_hDb, SQL_CheckCallback, szQuery, pack, DBPrio_Low);
+
+	}
+	else{
+		//PrintToConsole(client,"NEW ENTRY");
+
+		// Escape name for SQL injection protection
+		char szName[MAX_NAME_LENGTH * 2 + 1], szUName[MAX_NAME_LENGTH];
+		GetClientName(client, szUName, MAX_NAME_LENGTH);
+		SQL_EscapeString(g_hDb, szUName, szName, MAX_NAME_LENGTH);
+
+		int zonegroup = ReadPackCell(pack);
+
+		char szQuery[1024];
+
+		Format(szQuery, 1024, sql_insertPR, szSteamID, szName, szMapName, 0.0, zonegroup, 0.0, 0.0, 0.0, 0.0);
+		SQL_TQuery(g_hDb, SQL_CheckCallback, szQuery, pack, DBPrio_Low);
+
+	}
+
+	CloseHandle(pack);
+	return;
+}
+
+public void db_UpdatePRinfo_WithRuntime(int client, char szSteamID[32], int zGroup, float runtime)
+{	
+	if (!IsValidClient(client))
 		return;
-	}
+
+	char szQuery[2048];
+	//PrintToConsole(client, "%f || %f || %f || %f || %f\n", g_fTimeinZone[client][zGroup], g_fCompletes[client][zGroup], g_fAttempts[client][zGroup], g_fstComplete[client][zGroup], runtime);
+	Format(szQuery, 2048, sql_updatePrinfo_withruntime, g_fTimeinZone[client][zGroup], g_fCompletes[client], g_fAttempts[client][zGroup], g_fstComplete[client][zGroup], runtime, szSteamID, g_szMapName, zGroup);
+	SQL_TQuery(g_hDb, SQL_CheckCallback, szQuery, DBPrio_Low);
```

Comment:
you're using `DBPrio_Low` as data here instead of prio. You can use `.prio=DBPrio_Low` here.